rero_ils/modules/items/api.py

```diff
@@ -894,6 +894,15 @@ def checkout(self, current_loan, **kwargs):
         actions.update({LoanAction.CHECKOUT: loan})
         return self, actions
 
+    @add_loans_parameters_and_flush_indexes
+    def update_loan(self, current_loan, **kwargs):
+        """Update a loan."""
+        loan = Loan.get_record_by_pid(kwargs.get('pid'))
```

Comment:
Is it possible there is no `pid` in `kwargs`, in this case line 903 will not work.

Reply:
This PR is still a draft and not ready for review.